squarelet/organizations/serializers.py

```diff
@@ -7,12 +7,18 @@
 from rest_framework.exceptions import APIException
 
 # Squarelet
-from squarelet.organizations.models import Charge, Membership, Organization
+from squarelet.organizations.models import (
+    Charge,
+    Membership,
+    Organization,
+    OrganizationSubtype,
+)
 
 
 class OrganizationSerializer(serializers.ModelSerializer):
     uuid = serializers.UUIDField(required=False)
     merged = serializers.SlugRelatedField(read_only=True, slug_field="uuid")
+    subtypes = serializers.StringRelatedField(many=True)
```

Comment:
This could be a `SlugRelatedField` pointed to name if you want only the name to show up in the results - the `str` method for SubType includes the Type name, which I think looks nice.

Reply:
Yeah, I agree having the type name is helpful!